pkg/assets/kubernetes.go

```diff
@@ -0,0 +1,38 @@
+package assets
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/golang/glog"
+)
+
+var (
+	// kubeStaticAssets defines the source template (relative to manifest dir) and the path to output file (relative to asset dir).
+	kubeStaticAssets = []string{
+		// bootstrap manifests
+		"bootstrap-manifests/bootstrap-apiserver.yaml",
```

Comment:
is there a need to have this in a subdir?

Reply:
bootkube convention